Northstar.CustomServers/mod/scripts/vscripts/_loadouts_mp.gnut

```diff
@@ -62,20 +60,10 @@ void function SetLoadoutGracePeriodEnabled( bool enabled )
 
 void function SetWeaponDropsEnabled( bool enabled )
 {
-	file.weaponDropsEnabled = enabled
-}
-
-void function DestroyDroppedWeapon( entity victim, entity attacker, var damageInfo )
-{
-	if ( !file.weaponDropsEnabled && IsValid( victim.GetActiveWeapon() ) )
-		thread DelayDestroyDroppedWeapon( victim.GetActiveWeapon() )
-}
-
-void function DelayDestroyDroppedWeapon( entity weapon )
-{
-	WaitEndFrame()
-	if ( IsValid( weapon ) )
-		weapon.Destroy()
+	if( !enabled )
+		FlagSet("WeaponDropsAllowed")
```

Comment:
`FlagSet` basically sets the flag to true, right?  Or does it invert it based on what it was set to in `FlagInit`?

Either way, is this logic correct? `SetWeaponDropsEnabled( true )` would currently do `FlagClear`

Reply:
FlagSet sets it to true yes, also yeah now thinking abou it I shouldnt invert the enable anyways that just makes it worse to read, I will fix that in a sec. ( it was late for me too ^^ )